### ISSUE TITLE
Is it possible to call predict() from a bytes object instead of a file?

### ISSUE THREAD

**Issue by darrinmwiley** [spotify/basic-pitch#55]
It would be great if it was possible to feed data directly into the model instead of having to save it to a file and pass the path in.

**Comment by DarylWM** [comment]
I would like to see this as well. My preference would be a numpy 2D array of shape (samples, channels).

**Comment by energydrink9** [comment]
Yes, that would be very useful in order to integrate the tool in other applications

**Comment by Laubeee** [comment]
same.
The API change is quite small and would not break compatibility. In `get_audio_input()` librosa needs to call resample instead of load. The audio should, next to `str` and `Path` also support a tuple of (ndarray, int) representing samples and sample_rate.
